internal/provider/user_role_grants_resource_test.go

```diff
@@ -140,10 +162,11 @@ func testRoleResource(t *testing.T, userId string, useMock bool) {
 		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
 		Steps: []resource.TestStep{
 			{
-				Config: getTestRoleFullyPermissionsedResourceConfig(userId),
+				Config: getTestRoleFullyPermissionsedResourceConfig(),
 				Check: resource.ComposeTestCheckFunc(
-					testRoleMembership(resourceNameTest, userId, "CLUSTER_ADMIN", true),
-					resource.TestCheckResourceAttr(resourceNameTest, "id", userId),
+					testRoleMembership(resourceNameTest, "CLUSTER_ADMIN", true),
+					testRoleMembership(resourceNameTest, "ORG_MEMBER", true),
+					testRoleMembership(resourceNameTest, "ORG_ADMIN", true),
```

Comment:
checking that all roles are present

Reply:
I'm not sure. Checking all seems good to me.